src/index.html

```diff
@@ -11,6 +11,30 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="container">
+      <form action="#" method="get">
+        <div class="input-wrapper__first">
```

Comment:
```suggestion
      <form class="search_bar" data-qa="big" action="#" method="get">
```
please proofread task descrioption. data-qa="big" should be on form tag. div lools like redundant - add necessary styles for form (set some class for it)
each input should be wrapped in form tag.
![Screenshot from 2024-02-04 15-20-11](https://github.com/mate-academy/layout_search-bar-airbnb/assets/33029864/b8a40d6a-ef40-49e9-a2de-6c462149dd25)